.env.docker

```
@@ -9,3 +9,7 @@ DB_PORT=5432
 # PGADMIN admin user 
 PGADMIN_EMAIL=admin@admin.com
 PGADMIN_PASSWORD=root
+
+# Frontend
+ESID_PORTS=8080:80
```

This is misleading, because one might think you could change the port inside the container. But the nginx in the container is always bound to Port 80. So if somebody changes the 80 in the config, the request won't pass through to the nginx inside.

A better way is to just change the public facing port.
```suggestion
ESID_PORT=8080
```